src/types/events/base-events.ts

```diff
@@ -417,6 +418,26 @@ export interface IMOpenEvent extends StringIndexed {
   channel: string;
 }
 
+export interface InviteRequestedEvent extends StringIndexed {
+  type: 'invite_requested';
+  invite_request: {
+    id: string;
+    email: string;
+    date_created: number;
+    requester_ids: string[];
+    channel_ids: string[];
+    invite_type: string;
```

Comment:
there are three specific strings the `invite_type` value can be: `restricted`, `ultra_restricted`, or `full_member`. source: https://api.slack.com/admins/inviting#events

Reply:
@aoberoi Good catch! I just modified it to be more specific. 👍